lib/Elasticsearch/Tests/QueryConditionGeneratorTest.php

```diff
@@ -44,7 +44,7 @@ public function it_throws_a_logic_exception_if_referencing_a_field_not_in_fields
             ->field('unknown')
                 ->addSimpleValue('nope')
             ->end()
-        ->getSearchCondition();
+            ->getSearchCondition();
```

Comment:
Could you please remove these since they don't contribute to the solution here and just create noise in the PR.

Reply:
Done.